Tests/UnitTests/Plugins/TGeo/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+set(unittest_extra_libraries ActsTGeoPlugin)
+
+add_unittest(TGeoArb8ConversionTests TGeoArb8ConversionTests.cpp)
```

```suggestion
add_unittest(TGeoArb8Conversion TGeoArb8ConversionTests.cpp)
```
`add_unittest` already prefixes the resulting binary with `ActsUnitTest...`. Appending `...Tests` to the name results in a duplicated name `ActsUnitTest...Tests`. The `...Tests` can go.

---

Ok, no objection.

---

Fixed.